lib/rubycritic/rake_task.rb

```diff
@@ -0,0 +1,57 @@
+require "rake"
+require "rake/tasklib"
+
+module Rubycritic
+  class RakeTask < Rake::TaskLib
+    attr_accessor :name
+    attr_accessor :report
+    attr_accessor :format
+    attr_accessor :paths
+    attr_accessor :options
+
+    def initialize(*args, &task_block)
+      setup_ivars(args)
+
+      desc "Run rubycritic" unless ::Rake.application.last_comment
+
+      task(name, *args) do |_, task_args|
+        RakeFileUtils.send(:verbose, verbose) do
```

Comment:
It makes `rake` output all file operations it performs, but it isn't really needed here as there are none...